akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -254,6 +254,10 @@ sealed trait ActorPath extends Comparable[ActorPath] with Serializable {
  */
 @SerialVersionUID(1L)
 final case class RootActorPath(address: Address, name: String = "/") extends ActorPath {
+  require(name.length == 1 || name.indexOf('/', 1) == -1,
```

Comment:
oh...that will break a lot of stuff... I think you should put a note into migration guide